satellite/rangedloop.go

```diff
@@ -74,7 +75,9 @@ func NewRangedLoop(log *zap.Logger, full *identity.FullIdentity, db DB, metabase
 	}
 
 	{ // setup ranged loop
-		peer.RangedLoop.Service = rangedloop.NewService(log.Named("rangedloop"), config.RangedLoop, metabaseDB, nil)
+		// TODO: replace with real segment provider
+		segments := &rangedlooptest.RangeSplitterMock{}
```

Comment:
Isn't this an import cycle?

Reply:
It wouldn't compile if it was?

Reply:
I misread the filename at the top. I thought this was a file in the satellite/metabase/rangedloop package.